go.mod

```diff
@@ -5,12 +5,14 @@ go 1.14
 // replace github.com/openziti/foundation => ../foundation
 
 require (
+	github.com/alangpierce/go-forceexport v0.0.0-20160317203124-8f1d6941cd75 // indirect
 	github.com/emirpasic/gods v1.12.0
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.0
```

Comment:
Should protobufs be regenerated if we're updating to 1.4.0?

Reply:
Rolled it back to 1.3.5. Most likely came along when `go mod` decided to pull new stuff in.

Reply:
Need to hit this `foundation` PR before I can merge this `fabric` PR:

https://github.com/openziti/foundation/pull/113